### DISCUSSION TITLE
How does the auto update handle the portable version?

### DISCUSSION THREAD

**Discussion by lwcorp** [sandboxie-plus/Sandboxie#1930] [Documents and notes]
[v1.0.22](https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.0.22) "added auto update download and silent install option to sandman.exe".

Since this option appears in the portable version as well, how does it handle it?
Does it automatically:
1. Close sandman.exe
1. Export the new files into its folder
1. Re-opens sandman.exe

?

![image](https://user-images.githubusercontent.com/1773306/172037583-9d2fc6fe-b83c-402a-8a26-711f613816a0.png)

**Comment by DavidXanatos** [comment]
It downloads but does not install, you need to start install manually

**Comment by lwcorp** [comment]
Do you refer to the regular version too? If so, how do you trigger the aforementioned silent install?

If you specifically mean the portable version, does it at least launch the downloaded EXE file? If so, if you choose "only export" does it shut down sandman.exe before doing so?

**Reply by DavidXanatos** [reply]
you wil be shown a messagebox there you need to click on ok than everythign will be closed and the instal executed, dont do that for portable mode obviusly

**Reply by lwcorp** [reply]
So what did you mean that v1.0.22 added silent install?

And in the portable version why shouldn't it close it too and export the files there?

**Reply by DavidXanatos** [reply]
it does not show the instaler UI when started but it must be manually initiated

**Reply by lwcorp** [reply]
So:
1. In the regular version it silently installs.
1. In the portable version it just downloads a file, which the user needs to locate (after closing the program).

?

**Comment by DavidXanatos** [comment]
no boot versions behave same, you always get a messagebox asking if you want to run the downloaded setup

**Reply by lwcorp** [reply]
And in both versions if you choose yes does it automatically close the existing program?
